client/src/common/components/Form.jsx

```diff
@@ -4,6 +4,7 @@ import { FormattedMessage, injectIntl } from 'react-intl';
 
 import { ListType } from 'modules/list/consts';
 import { IntlPropType } from 'common/constants/propTypes';
+import './Form.scss';
```

Comment:
Please split external and internal import groups.

Reply:
Here imports are split. `import { IntlPropType } from 'common/constants/propTypes';` is our internal import